lib/desired-caps.js

```diff
@@ -0,0 +1,14 @@
+let geckodriverConstraints = {
+  geckodriverPort: {
+    isNumber: true
+  },
+  // duplicate of above with better spelling
```

Comment:
If we don't do this for the matter of being backwards compatible I wouldn't do this especially since Appium errors out properly and users will know what to change.

Reply:
Sounds good! I was copy-pasting this from appium-chromdriver 😃